pkg/kubelet/kubelet_pods.go

```diff
@@ -138,6 +139,11 @@ func makeMounts(pod *v1.Pod, podDir string, container *v1.Container, hostName, h
 			return nil, err
 		}
 		if mount.SubPath != "" {
+			err = volumevalidation.ValidatePathNoBacksteps(mount.SubPath)
```

Comment:
since `IsAbs()` is also platform-dependant, should verify `!filepath.IsAbs(mount.SubPath)` here